### DISCUSSION TITLE
Seeing Aborted Processe with errorStrategy = 'finish'

### DISCUSSION THREAD

**Discussion by bgobbi** [nextflow-io/nextflow#1959] [Q&A]
I am seeing that after an error of one task other parallel executed tasks running on slurm are aborted even though I specified `errorStrategy = 'finish'` for the given process.
Is this expected?

Also a related question. I would like my errorStrategy to be that all tasks that do not depend on the failed task are executed and run to completion. I have raid the [documentation](https://www.nextflow.io/docs/latest/process.html#errorstrategy) and none of the implemented strategies seems to do exactly that.
Is there a way?

Thanks!

**Comment by abhi18av** [comment]
Hi @bgobbi 

> I would like my errorStrategy to be that all tasks that do not depend on the failed task are executed and run to completion ...

For accomplishing this, it might be worth exploring `errorStrategy = 'ignore'` option.

**Reply by bgobbi** [reply]
Thanks! @abhi18av .
I would prefer the pipeline to stop executing new tasks but finish existing ones. Lets say i realized that many jobs will run out of memory but some do not. Killing the few ones that are still running and will probably give good results would be a waste of those resources. Continuing would also waist the resources of the jobst that will start but fail later.
My impression was that ```errorStrategy = 'finish'``` was meant to do exactly that.
